lib/challenge.js

```diff
@@ -61,6 +91,24 @@ var some_pet;
 var Pet;
 // Answer Starts Here
 
+var Pet = function Pet(name, species, breed, noise) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = []];
+
+};
+
+Pet.prototype.makeNoise = function makeNoise () {
```

Comment:
Awesome. This question was asking you to create a constructor function and attach a method to its prototype, which you did.